text/src/autogluon/text/automm/predictor.py

```diff
@@ -355,7 +386,7 @@ def fit(
             assert self._output_shape == output_shape, \
                 f"Inferred output shape {output_shape} is different from " \
                 f"the previous {self._output_shape}"
-
+                
         if self._df_preprocessor is None:
             df_preprocessor = init_df_preprocessor(
```

Comment:
We may need to move `init_df_preprocessor` into `_fit()` since users may need to tune some hyper-parameters in config.data, e.g., `convert_to_text`. Here we only need to create and fit the LabelEncoder.